kitsune/users/forms.py

```diff
@@ -88,54 +75,6 @@ def save_for_user(self, user):
                 user.settings.create(name=field, value=value)
 
 
-class AuthenticationForm(auth_forms.AuthenticationForm):
```

Comment:
There's a reference to this form in ModelBackendAllowInactive:
```
Standard model authentication that also allows inactive users.

    This is necessary as new registered users are marked inactive
    but still logged in automatically. Some control around logging
    in as an active user is still retained via the ``only_active``
    argument of ``kitsune.users.forms.AuthenticationForm``.
```
Is it still true that new registered users are marked inactive? This might be something we can tear out later.

Reply:
New users are not marked inactive. Active/Inacative users is one of the things this PR does not solve. I will have a look at that reference though- Thanks

Reply:
I think that means we can remove `ModelBackendAllowInactive` then.